WordPress/Classes/Extensions/Colors and Styles/WPStyleGuide+Search.swift

```diff
@@ -1,19 +1,25 @@
 import Foundation
 import WordPressShared
+import UIKit
 
 extension WPStyleGuide {
 
     fileprivate static let barTintColor: UIColor = .neutral(.shade10)
 
-    @objc public class func configureSearchBar(_ searchBar: UISearchBar) {
+    public class func configureSearchBar(_ searchBar: UISearchBar, backgroundColor: UIColor, returnKeyType: UIReturnKeyType) {
         searchBar.accessibilityIdentifier = "Search"
         searchBar.autocapitalizationType = .none
```

Comment:
Should we enable this? I would think most users would expect auto capitalization for a site title.

Reply:
If we add any form of autocapitalization, we are making assumptions on how users want their site name, which we don't really know (what if I only want lowercase?).

Reply:
That's a fair point, and I'd think we'd need to factor in what most users expect here. This being a website title, my personal hunch would be to capitalize the first letter.

Perhaps we could get @kyleaparker's input? This is a non-blocker for the PR. Thanks!

Reply:
I would also expect the first letter to be capitalized for a site title. I can definitely see some people wanting all lowercase, but I would assume it's more common to want the first letter capitalized

Reply:
> I would also expect the first letter to be capitalized for a site title. I can definitely see some people wanting all lowercase, but I would assume it's more common to want the first letter capitalized

As a matter of fact, I just checked and in the existing text field to customize the site title (the one that appears when you tap on the site title in My Site), we do capitalize the first letter (and any letter that comes after a period). So, for consistency, I'll change the option to `.sentences`

Reply:
Great thought - thanks for testing that @Gio2018!